doc/troubleshooting_alerts.md

```diff
@@ -0,0 +1,101 @@
+# Troubleshooting Alerts
+
+You've been pinged by New Relic. Don't panic! Here are the next steps to
+take to work out what's going on, who to tell and how to fix it.
+
+## Jump into Slack
+
+You're likely not the only person who's seen the alert. Coordination will
+happen on Slack, so get on there first and see what people have ascertained
+already, to avoid duplication of work and stepping on each others' toes.
+
+## Acknowledge the alert
+
+The New Relic incident page (or the page on the mobile app) should have a
+big **Acknowledge** button. **Only press it once you're certain that the
+alert is being worked on** (by you or someone else).
+
+## Remember to close the NR incident afterwards
+
+This may seem a little early to mention it, but it's **really important**:
```

Comment:
So maybe just move this to the end?

And possibly at the beginning, a new very concise summary of the process, so that this is mentioned. E.g.,

> In short, we (1) Do X, (2) Do Y, and finally, we (3) close the NR incident.